features/steps/partcad-cli/commands/init.py

```
@@ -94,6 +94,9 @@ def step_impl(context):
             if yaml_content is None:
                 raise ValueError("YAML content is empty or invalid")
 
+            if yaml_content.get("private") is not True:
+                raise ValueError("The package must be marked as private")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)